src/util.h

```diff
@@ -13,9 +13,12 @@
 
 #include <stdlib.h>
 #include <stdint.h>
-#include <stdio.h>
 #include <limits.h>
 
+#ifndef USE_EXTERNAL_DEFAULT_CALLBACKS
+#include <stdio.h>              /* for fprintf */
```

Comment:
```suggestion
#include <stdio.h>
```

style nit: we usually don't say for what we need includes. (And it's not only needed for `fprintf`, it's also needed for `stderr`. )

Reply:
Sure thing, thanks.